### ISSUE TITLE
How do I use the generated images vector sequences for portraying real images?

### ISSUE THREAD

**Issue by hunkim98** [CyberAgentAILab/canvas-vae#5]
Thank you for creating this code!

I managed to successfully train the PixelVAE model and CanvasVAE model, and I have questions regarding how to plot a reconstructed layout as a real banner image.

As far as I know, the PixelVAE model encodes pillow images into vectors (512 dim) and the CanvasVAE model interprets the image sequences as a collection of those vectors.

The output of the CanvasVAE, are all in numbers (whether they are discretized values or float numbers), and while it is possible to portray the generated outputs as bounding boxes using the top, left, width, and height values, I am not sure how to convert image vector sequences to real rgba images.

I could use the PixelVAE decoder, but the outputs are too blurry and hard to distinguish (hardly usable as real images)

In your paper, I read that you have randomly generated crello documents with real images portrayed in the banners. Is there a specific method you used to generate real crisp images based on the generated image vector?

**Comment by kyamagu** [comment]
@hunkim98 The images are retrieved from the training dataset in the paper. See Section 5.4

**Comment by hunkim98** [comment]
Oh, I see! the images and shape seem to have been filled using a "nearest neighbor retrieval" on the existing images and shapes. Thanks for your clarification!
